src/components/Book.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => (
+  <tr>
+    <td>{book.id}</td>
+    <td>{book.title}</td>
+    <td>{book.category}</td>
+  </tr>
+);
+
+export default Book;
+
+Book.propTypes = {
+  book: PropTypes.object.isRequired,
```

Comment:
How about describing the exact object shape needed to avoid object without some properties being passed down to this component